src/sentry/utils/safe.py

```diff
@@ -115,21 +115,37 @@ def get_path(data: PathSearchable, *path, **kwargs):
     filtered with the given callback. Alternatively, pass ``True`` as filter to
     only filter ``None`` values.
     """
+    logger = logging.getLogger(__name__)
     default = kwargs.pop("default", None)
     f: Optional[bool] = kwargs.pop("filter", None)
     for k in kwargs:
         raise TypeError("get_path() got an undefined keyword argument '%s'" % k)
 
+    logger_data = {
+        "path_searchable": json.dumps(data),
```

Comment:
Hmm, looking at this again, I'm not sure if it's a good idea to add this logging here. `get_path` is used in many places including ingestion, and data could be quite large. It might be expensive to be dumping the whole thing. If we want this logging, could we maybe only do this logging if we pass some parameter?

Reply:
fair point, i can add a conditional since we should only really care about the logs for when the path parameter consists of ["totals", "rate"]

Reply:
or maybe to make it easier i can add an optional bool to indicate whether to log or not that defaults to false